CHANGES.txt

```diff
@@ -68,6 +68,20 @@
 * Allow combination of pip list options --editable with --outdated/--updtodate.
   (:issue:`933`)
 
+* Gives VCS implementations control over saying whether a project 
+  is under their control (:pull:`3258`)
+
+* GIT detection now works when ``setup.py`` is not at the GIT repo root
```

Comment:
Nit-picking a little but the correct case seems to be `Git`, cf https://git-scm.com/